core/sections/clause_8_vector.adoc

```diff
@@ -18,25 +19,44 @@
 
 * color: Color
 
-==== Class Stroke
-===== Class GraphicalUnit
-===== Class StrokeStyling
+==== Class Graphic
 
-==== Class MultiGraphic
-===== Class Marker
+* opacity: float
+* transform: Transform2D
 
-==== Class Graphic
 ===== Class Image
-====== Class Resource
-===== Class Text
-The text property will exceptionnally support an identifier parameter value expression (without any additional conformance class required).
 
-====== Class Font
-====== Class TextAlignment
+* image: Resource
+* hotSpot: Pointf
+* tint: Color
+* blackTint: Color
+* alphaThreshold: float
 
-* enum HAlignment
-* enum VAlignment
+===== Class Text
+
+* text: string
+* font: Font
+* alignment: TextAlignment
 
 ===== Class Shape
+
 ====== Class Dot
-* point: Pointf
+
+===== Class MultiGraphic
+
+* elements: Graphic [0..*]
+
+====== Class Marker
+
+==== Class Resource
```

Comment:
Resource should be under the Image section where it is used, with its properties.